apps/condo/domains/notification/utils/serverSchema/messageTypesHooks/NewsItemCommonHooks.js

```diff
@@ -0,0 +1,28 @@
+const dayjs = require('dayjs')
+const get = require('lodash/get')
+
+const { count } = require('@open-condo/keystone/schema')
+
+const { AbstractHooks } = require('./AbstractMessageTypesHooks')
+
+class NewsItemCommonHooks extends AbstractHooks {
+
+    async shouldSend () {
+        const userId = get(this.messageAttrs, ['user', 'connect', 'id'])
+        const messageType = get(this.messageAttrs, 'type')
+
+        if (!userId || !messageType) {
```

Comment:
Absence of user is not sufficient way to check if notification is of push type. With adr-7 there was implemented ability to send sms/emails not only to phone/email but to user's phone/email also.

Reply:
This code was removed after refactoring.